templates/cloud_sidebar.hbs

```diff
@@ -57,6 +57,8 @@
                     <li><a {{#if (active_content request.spin-path-info "/cloud/spin-vs-code-extension" )}}
                             class="active" {{/if}} href="{{site.info.base_url}}/cloud/spin-vs-code-extension">Using VS
                             Code Extension</a></li>
+                    <li><a {{#if (active_content request.spin-path-info "/cloud/noops-sql-db" )}} class="active"
+                            {{/if}} href="{{site.info.base_url}}/cloud/noops-sql-db">Persisting Data With NoOps SQL Database</a></li>
```

Comment:
Again, I'm not sure that this belongs in the tutorials section but it's a bit of a queer duck so 🤷

Reply:
The other related "Persisting Data With ..." are also in the tutorial section and this can fit there. The word "tutorial" has been removed from the page, to match the other related pages. (Also looking at the other heading sections "Concept", "References", "Support" it is less compatible with those; so here works.)

Reply:
I am happy with this for now.